e2etest/newe2e_config.go

```diff
@@ -50,10 +50,8 @@ type NewE2EConfig struct {
 		} `env:",required"`
 
 		StaticStgAcctInfo struct {
-			StaticOAuth struct {
```

Comment:
nit: this could be done as an either/or setup

weigh this with a grain of salt, most of the team will probably do inherited auth

Reply:
on a similar note, could it be set up to allow AzCLI or AzPwsh inheritance? Some people on the team use one or the other, not usually both

Reply:
Let's add the option of Pwsh as a separate PR just to keep things simple now. @adreed-msft could you create an item for that under AzCopy as a library?